RUFAS/biophysical/animal/digestive_system/enteric_methane_calculator.py

```diff
@@ -0,0 +1,337 @@
+        float
+            The amount of methane emission for calf (g/day).
+
+        """
+        methane_emission = 0.0
+        if methane_model:
+            methane_emission = (0.013 * (body_weight**0.75) * 4.184) / 0.05565
+
+        return methane_emission
+
+    @staticmethod
+    def calculate_heifer_methane(
+        methane_model: str | None, dry_matter_intake: float, nutrient_concentrations: dict[str, float]
+    ) -> float:
+        """
+        Calculates the amount of methane emission for heifer.
+
+        Parameters
+        ----------
+        methane_model: str
+            Methane model used for methane emission calculations, including Boadi, IPCC.
+        dry_matter_intake: float
+            Amount of dry matter intake (kg).
+        nutrient_concentrations: dict[str, float]
+            Concentrations of nutrients in pen ration, calculated per animal, percentages.
+
+        Returns
+        -------
+        float
+            Amount of methane emission for heifer (g/day).
+
+        """
+        methane_emission = 0.0
+        if methane_model:
+            # Default: IPCC Tier 2
+            crude_protein_concentration = nutrient_concentrations["CP"]
+            ethyl_ester_concentration = nutrient_concentrations["EE"]
+            neutral_detergent_fiber_concentration = nutrient_concentrations["NDF"]
+            ash_concentration = nutrient_concentrations["ash"]
+            soluble_residue = (
+                (100 - ash_concentration)
+                - neutral_detergent_fiber_concentration
+                - crude_protein_concentration
+                - ethyl_ester_concentration
+            )
+            gross_energy_concentration = (
+                0.263 * crude_protein_concentration
+                + 0.522 * ethyl_ester_concentration
+                + 0.198 * neutral_detergent_fiber_concentration
+                + 0.160 * soluble_residue
+            )  # [A.3B.C.2]
+            methane_emission = (0.065 * gross_energy_concentration * dry_matter_intake) / 0.05565  # [A.3B.C.3]
+
+        return methane_emission
+
+    @staticmethod
+    def calculate_cow_methane(
+        is_lactating: bool,
+        body_weight: float,
+        milk_fat: float,
+        metabolizable_energy_intake: float,
+        nutrient_amounts: dict[str, float],
+        nutrient_concentrations: dict[str, float],
+        methane_mitigation_method: str,
+        methane_mitigation_additive_amount: float,
+        methane_model: str,
+    ) -> float:
+        """
+        Calculates the daily enteric emissions for cows.
+
+        Parameters
+        ----------
+        body_weight: float
+            Body weight of the current cow (kg).
+        methane_model: str
+            Methane model used for methane emission calculations, including "Mutian", "Mills", "IPCC".
+        is_lactating: bool
+            Indicator of cow's lactating status.
+        milk_fat: float
+            Milk fat (from animal input), % of milk.
```

Comment:
```suggestion
            Milk fat, % of milk.
```
Right now this value comes from the animal input, but this may not always be the case. And because the source of this value doesn't change how this method works, I don't think it is necessary to include it.